alias.sh

```diff
@@ -0,0 +1,44 @@
+#!/bin/sh
+
+# Configuration file containing shortened versions and associated commands
+config_file='alias-config.txt'
+
+# Check if the config file exists
+if [ ! -f "$config_file" ]; then
+  echo "Configuration file not found: $config_file"
+  exit 1
+fi
+
+# Function to display help message
+show_help() {
+  echo "Usage: $0 <alias>"
+  echo ""
+  echo "Options:"
+  echo "  -h   Show help message"
+  echo ""
+  echo "Available commands:"
+  while IFS=':' read -r alias command || [ -n "$alias" ]; do
+    printf "  %-8s %s\n" "$alias" "$command"
+  done < "$config_file"
+}
+
+# If no arguments provided or the argument is -h, show help message
+if [ $# -eq 0 ] || [ "$1" = "-h" ]; then
+  show_help
+  exit 0
+fi
+
+# Read the config file to find the matching shortened version
+while IFS=':' read -r alias command; do
+  if [ "$alias" = "$1" ]; then
+    # The -q flag suppresses the output from the script command itself
+    # The -c flag specifies the command to run and /dev/null discards the typescript file that script would normally generate
+    script -q -c "$command" /dev/null
```

Comment:
I'm not exactly sure for the usecase of this. Is there a reason why we are not going directly to $command?

`script` also does not have the -c flag on my machine, so this does not seem to work out of the box for some developers. Can we make this script more friendly for more machines? That said, I don't think we should support windows, as it looks like no one in the team use windows. Thoughts?

Reply:
If you "go directly to $command" the script will exit with `the input device is not a TTY` 

You can get around the TTY by passing `-T` option to "allocate a psuedo-TTY" but then the output in terminal looks a little different. So like for `./alias.sh test` it would look like this instead of having the nice default colors:

<details><summary>See screenshot</summary>

![image](https://user-images.githubusercontent.com/73561520/236644193-b68f1f72-437e-4f13-bb83-8b9bcf7cfbed.png)

</details>

But perhaps I'm too attached to the colors and psuedo-TTY output is fine?

Reply:
I'm a bit confused. Are we seeing the same thing? What is the program you are using to run your clis?

<img width="800" alt="Screenshot 2023-05-06 at 3 05 00 PM" src="https://user-images.githubusercontent.com/62368440/236648300-d4428d68-a908-469c-b510-eabe2046e0a0.png">
<img width="800" alt="Screenshot 2023-05-06 at 3 05 23 PM" src="https://user-images.githubusercontent.com/62368440/236648302-abfb4bf1-8dfd-42d8-a4c2-98020abd7bc3.png">

Reply:
Ahh, I see this is happening for the test and lint commands. Interesting.

Reply:
That said it looks like this configuration works out of the box with the behavior that we would expect it to. Any thoughts on whether this is a viable path forward? 

<img width="800" alt="Screenshot 2023-05-06 at 3 14 59 PM" src="https://user-images.githubusercontent.com/62368440/236648583-01389136-24ff-407d-aba5-ba9af1724116.png">

Reply:
> What is the program you are using to run your clis?

Ubuntu runs `.sh` files with with the `bash` executable found in `/usr/bin/bash` which I beleive is the same for all Unix-based systems including macOS

> Any thoughts on whether this is a viable path forward?

Looks like swapping `compose run` for `compose up` is viable for the `lint` but not the `test` command.

![image](https://user-images.githubusercontent.com/73561520/236956031-5edee387-56be-44db-b0e8-ad6f092d8a8e.png)

However, we can still run the `test` command with `test:docker compose run -T webpack npm run` 

![image](https://user-images.githubusercontent.com/73561520/236957309-8795e082-61c3-4f0d-91ce-6d97f79c6451.png)